### PR TITLE
zh-translation:/faq/mixer/attribute-expressions.md

### PR DESCRIPTION
**Pull request by TomatoAres**
ref: https://github.com/servicemesher/istio-official-translation/issues/1414

### CODE REVIEW

File: content/zh/faq/mixer/attribute-expressions.md [hunk 1]
```diff
@@ -1,8 +1,6 @@
 ---
-title: What is the full set of attribute expressions Mixer supports?
+title: MIXER 支持的全部属性表达式有哪些？
```
Comment (rootsongjc) [inline content/zh/faq/mixer/attribute-expressions.md]: MIXER -> Mixer

File: content/zh/faq/mixer/attribute-expressions.md [hunk 2]
```diff
@@ -1,8 +1,6 @@
...
 weight: 20
 ---
 
-Please see the [Expression Language
-Reference](/docs/reference/config/policy-and-telemetry/expression-language/) for
-the full set of supported attribute expressions.
+Mixer 支持的全部属性表达式参见[表达式语言参考](/docs/reference/config/policy-and-telemetry/expression-language/)。
```
Comment (rootsongjc) [inline content/zh/faq/mixer/attribute-expressions.md]: 请在链接前增加中文 `/zh` 前缀。

**Comment by TomatoAres** [comment]
@googlebot I fixed it.

**Pull request merged**